src/main/java/org/aion/wallet/account/AccountManager.java

```diff
@@ -77,14 +78,18 @@ private void registerEventBusConsumer() {
         EventBusFactory.getBus(SettingsEvent.ID).register(this);
     }
 
-    public String  createAccount(final String password, final String name) {
+    public String createAccount(final String password, final String name) throws ValidationException {
         final StringBuilder mnemonicBuilder = new StringBuilder();
         final byte[] entropy = new byte[Words.TWELVE.byteLength()];
         new SecureRandom().nextBytes(entropy);
         new MnemonicGenerator(English.INSTANCE).createMnemonic(entropy, mnemonicBuilder::append);
         final String mnemonic = mnemonicBuilder.toString();
-        final byte[] seed = new SeedCalculator().calculateSeed(mnemonic, DEFAULT_MNEMONIC_SALT);
+        final byte[] seed = getNewAccountSeed(mnemonic);
         final ECKey ecKey = new SeededECKeyEd25519(seed);
+        if (Keystore.exist(ByteUtil.toHexString(ecKey.getAddress()))) {
```

Comment:
validation is not required here so we should also not throw exception at method signature